e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -189,15 +189,15 @@ describe("viem send tx", () => {
 		assert.equal(receipt.status, "success", "receipt status 'failure'");
 	}).timeout(10_000);
 
-	it.skip("send overlapping nonce tx in different currencies", async () => {
+	it("send overlapping nonce tx in different currencies", async () => {
 		const priceBump = 1.1;
 
 		const rate = await getRate(process.env.FEE_CURRENCY);
 		// Native to FEE_CURRENCY
 		const nativeCap = 30_000_000_000;
 		const bumpCurrencyCap = rate.toFeeCurrency(BigInt(Math.round(nativeCap * priceBump)));
 		const failToBumpCurrencyCap = rate.toFeeCurrency(BigInt(
-			Math.round(nativeCap * priceBump) - 1,
+			Math.round(nativeCap / priceBump)
```

Comment:
I think the goal of this test is really to check that price bumps in alternate currencies are performed using the correct exchange rates. So to test this we want to ensure that our replacement transaction prices are close to the threshold which is `oldPrice * 1.1` so that incorrect exchange logic is more likely to result in a failures.

So I think dividing by priceBump is too extreme and instead I'd suggest multiplying by `1.09` to get something close to the threshold but below it. Same goes for below.